homeassistant/components/hive/climate.py

```diff
@@ -15,8 +17,10 @@
     SUPPORT_TARGET_TEMPERATURE,
 )
 from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS, TEMP_FAHRENHEIT
+from homeassistant.helpers import config_validation as cv, entity_platform
 
-from . import ATTR_AVAILABLE, DATA_HIVE, DOMAIN, HiveEntity, refresh_system
+from . import HiveEntity, refresh_system
+from .const import ATTR_AVAILABLE, ATTR_TIME_PERIOD, DOMAIN, SERVICE_BOOST_HEATING
```

Comment:
What is `ATTR_AVAILABLE` used for? What does that measurement mean?

Reply:
This attribute returns a string either online or offline its used as a device state attribute

Reply:
We should set the `available` Entity property instead.

https://developers.home-assistant.io/docs/core/entity#generic-properties

Reply:
The available property is in the code. Would some one be able to create an automation though using available? e.g if device is unavailable then send a notification?

Reply:
When `available` property returns `False` the entity will get state `"unavailable"`.

Reply:
ok I will remove